ofl/notosansduployan/article/ARTICLE.en_us.html

```diff
@@ -1,26 +1,5 @@
-<p>
-  Noto Sans Duployan is an unmodulated (“sans serif”) design for texts in the
-  American <em>Duployan shorthand</em> script.
-</p>
-<p>
-  Noto Sans Duployan contains 158 glyphs, and supports 154 characters from the
-  Unicode block Duployan.
-</p>
+<p>Noto Sans Duployan is a design for the <em>Duployan shorthand</em> script.</p>
+<p>Noto Sans Duployan contains 10,255 glyphs, 11 OpenType features, and supports 210 characters from 5 Unicode blocks: Duployan, Basic Latin, Combining Diacritical Marks, General Punctuation, Latin-1 Supplement.</p>
 <h3>Supported writing systems</h3>
 <h4>Duployan shorthand</h4>
-<p>
-  Duployan shorthand (Sloan-Duployan shorthand, Duployan stenography) is an
-  American alphabet, written left-to-right. Geometric stenography script created
-  in 1860 by Father Émile Duployé for writing French, later expanded and adapted
-  for writing English, German, Spanish, Romanian, and Chinook Jargon. Heavily
-  cursive (connected), allows words to be written in a single stroke. Praised
-  for simplicity and speed of writing. Needs software support for complex text
-  layout (shaping). Read more on
-  <a href="https://scriptsource.org/scr/Dupl">ScriptSource</a>,
-  <a href="https://www.unicode.org/versions/Unicode13.0.0/ch21.pdf#G27643"
-    >Unicode</a
-  >, <a href="https://en.wikipedia.org/wiki/ISO_15924:Dupl">Wikipedia</a>,
-  <a href="https://en.wiktionary.org/wiki/Category:Duployan_script"
-    >Wiktionary</a
-  >, <a href="https://r12a.github.io/scripts/links?iso=Dupl">r12a</a>.
-</p>
+<p>Duployan shorthand (Sloan-Duployan shorthand, Duployan stenography) is an shorthand alphabet, written left-to-right. Geometric stenography script created in 1860 by Father Émile Duployé for writing French, later expanded and adapted for writing English, German, Spanish, Romanian, and Chinook Jargon. Heavily cursive (connected), allows words to be written in a single stroke. Praised for simplicity and speed of writing. Needs software support for complex text layout (shaping). Read more on <a href="https://scriptsource.org/scr/Dupl">ScriptSource</a>, <a href="https://www.unicode.org/versions/Unicode15.0.0/ch21.pdf">Unicode</a>, <a href="https://en.wikipedia.org/wiki/ISO_15924:Dupl">Wikipedia</a>, <a href="https://en.wiktionary.org/wiki/Category:Duployan_script">Wiktionary</a>, <a href="https://r12a.github.io/scripts/links?iso=Dupl">r12a</a>.</p>
```

Comment:
Duployan was adapted for so many more languages than you have listed here. A better representative (but nowhere near exhaustive) list would be Chinook Jargon, English, Romanian, and Turkish, but it might be best to leave it vague without listing any specific languages.